cmd/vela-worker/operate.go

```diff
@@ -26,11 +28,14 @@ func (w *Worker) operate() error {
 	if err != nil {
 		return err
 	}
+	// ensure the address has one and only instance of "://"
+	addr := fmt.Sprintf("%s://%s", strings.TrimSuffix(w.Config.API.Protocol, "://"), w.Config.Hostname)
```

Comment:
TIL [strings.TrimSuffix](https://golang.org/pkg/strings/#TrimSuffix) - for Java I would have done a .replace on that char sequence with an empty string, but this is a prettier way of that!